tutorial/source/ss-vae.ipynb

```diff
@@ -300,7 +300,8 @@
     "def model_classify(xs, ys):\n",
     "    # this yields the extra term in the objective function\n",
     "    alpha = encoder_y.forward(xs)\n",
-    "    pyro.observe(\"y_aux\", dist.OneHotCategorical(alpha, log_pdf_mask=aux_loss_multiplier), ys)\n",
+    "    with poutine.scale(None, aux_loss_multiplier):\n",
+    "        pyro.observe(\"y_aux\", dist.OneHotCategorical(alpha), ys)\n",
```

Comment:
didnt realize this tutorial was still using `observe()`.. probably want to fix that in the tutorials and examples at some point